README.md

```diff
@@ -16,6 +16,11 @@ as well as deobfuscate scripts processed by
 # Usage
 You can beautify javascript using JS Beautifier in your web browser, or on the command-line using node.js or python.
 
+To use in web browser include the script tag below in your document
+```html
+<script src="https://cdn.rawgit.com/beautify-web/js-beautify/master/js/lib/beautify-html.js"></script>
```

Comment:
This will not really work fully.  People will need the css and js beautifiers as well.